.github/workflows/test-integrations.yml

```diff
@@ -536,6 +536,7 @@ jobs:
             -tags "${{ env.GOTAGS }}" \
             -timeout=20m \
             -parallel=2 \
+            -failfast \
```

Comment:
failures are super wasteful of time and resources because of the long waits and retries in all these tests. if there are flakes, we need to address them immediately, because they are particularly bad in this type of test.